mystore/src/main/java/be/cbtw/mystore/service/ClientService.java

```diff
@@ -0,0 +1,61 @@
+package be.cbtw.mystore.service;
+
+import be.cbtw.mystore.entity.Client;
+import be.cbtw.mystore.repository.ClientRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ClientService {
+
+    private final ClientRepository clientRepository;
+
+    public List<Client> getAllClients() {
+        return clientRepository.findAll();
+    }
+
+    public Client getClientById(Integer id) {
+        Optional<Client> client = clientRepository.findById(id);
```

Comment:
we can replace the whole function to:
`return clientRepository.findById(id).orElseThrow(Exception::new);`

For the next commit, we can create a custom exception for clients.